charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -340,12 +399,29 @@ spec:
       - name: kube-apiserver-admission-config
         configMap:
           name: kube-apiserver-admission-config
+      {{- if .Values.konnectivityTunnel.enabled }}
+      - name: konnectivity-server-certs
+        secret:
+          secretName: konnectivity-server
+      - name: konnectivity-server-kubeconfig
+        secret:
+          secretName: konnectivity-server-kubeconfig
+      - name: konnectivity-uds
+        emptyDir: {}
+      - name: egress-selection-config
+        configMap:
+          name: kube-apiserver-egress-selector-configuration
+      - name: konnectivity-server-log
```

Comment:
Hm, I have to admit that I'm confused in a similar way like @mvladev. Shouldn't we rely on the default log rotation and the logging stack deployed to the seed?

Reply:
What is the point of confusion here? you prefer logging to stdout as before? This changes was prompted due to https://github.com/gardener/gardener/pull/2251#issuecomment-638939076

Reply:
I'm wondering why you want to treat the logs of this container specially (compared to all other containers that we run in the seed/shoot).

Reply:
if I revert that change, lots of irrelevant useless logs would be seen in the log output which is undesired in my opinion.

Reply:
OK, fine then. If I remember correctly @mvladev mentioned that newer, not yet released, versions of the konnectivity will improve this behaviour. I'm okay with keeping it like this for now. Still, I think this is something that should be reconsidered before promoting the feature from its current alpha state.

Reply:
yes, since there is no verbosity level supported yet. Once verbosity levels are supported, we can remove that field and add a --v flag with important info.